plugins/aws/iam/iamRolePolicies.js

```diff
@@ -69,7 +69,15 @@ module.exports = {
             description: 'Ignores roles that contain the provided tag. Give key-value pair i.e. env:Finance ',
             regex: '^.*$',
             default: ''
-        }
+        },
+        resource_specific_wildcards: {
+            name: 'Service Specific Wildcards',
+            description: 'Resource specific wildcards in attached policies (inline and managed) which matches this regex. '+
```

Comment:
can we make this more descriptive? like:
input to flag resources based on regex. All policy resource which not pass this regex will be flagged.